interfaces/automated-testing/scd/scd_injection_observation.yaml

```diff
@@ -0,0 +1,792 @@
+        area_of_interest:
+          type: object
+          $ref: '#/components/schemas/Volume4D'
+                  
+      
+    SCDProcessingResults: 
+      description: Results for the operational intents submitted to the create test endpoint
+      type: array
+      items:
+        anyOf:
+          - $ref: '#/components/schemas/OperationalIntent'
+          - $ref: '#/components/schemas/FlightNotAuthorizedResult'
+
+    QuerySCDParameters:
+      description: |-
+        Parameters for a request to find results of strategic de-confliction processing matching the provided criteria.
+      type: object
+      properties:
+        area_of_interest:
+          $ref: '#/components/schemas/Volume4D'
+
+
+
+paths:
+  /scd_tests/{test_id}:
+    parameters:
+      -
+        examples:
+          default:
+            value: '2979bd18-7f06-441c-bda6-e82c841c35d6'
+        name: test_id
+        description: >-
+          The ID of the test.
+        schema:
+          type: string
+        in: query
+        required: true
+    put:
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateTestParameters'
+        required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CreateTestParameters'
+          description: Requested test was created successfully.
+        '409':
+          description: Test already exists.
+      operationId: createTest
+      summary: Create a test
+      description: >-
+        Request the creation of one or more operational intents based on injection of the provided Volume4D data.
+        These fields may not be modified.
+      security:
+        - TestAuth:
+            - scd.inject_test_data
+      tags:
+        - Strategic Deconfliction
+
+  /scd_tests/{test_id}/{version}:
+    parameters:
+      -
+        examples:
+          default:
+            value: 'b8e57460-f72d-4a8c-bb4e-b5ce1f5009c4'
+        name: test_id
+        description: >-
+          The ID of the test.
+        schema:
+          type: string
+        in: query
+        required: true
+      -
+        name: version
+        description: >-
+          Current version of the test.
+        schema:
+          type: string
+        in: query
+        required: true
+    delete:
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DeleteTestResponse'
+          description: Requested test was deleted successfully.
+      operationId: deleteTest
+      summary: Delete test
+      description: Remove all injected test data associated with this test from the host Service Provider's system under test.
+      security:
+        - TestAuth:
+            - scd.inject_test_data
+            
+      tags:
+        - Strategic Deconfliction
+
+  /scd_qualifier_data:    
```

Comment:
I think we would need to see the automated testing concept overall (perhaps with sequence and/or architecture diagram) to be sure, but I wouldn't expect this endpoint would be necessary.  In RID, we need an explicit observation endpoint because there is no standard API between the USS and Display Application/user.  In SCD, I don't think there is any corresponding interface that we want to tap into -- instead, the SCD automated testing can simply check with the DSS and USS directly using the ASTM-standard endpoints to determine if an operational intent was created correctly.

Reply:
OK understood, I will remove it from this review and in the test create response we can have the results of the creation request itself